feat: 엔티티 및 베이스 엔티티 클래스 추가

bsko98 opened the pull request:
## 📋 상세 설명

- 엔티티 및 베이스 엔티티 클래스 추가

## 📊 체크리스트

- [x] PR 제목이 형식에 맞나요 e.g. feat: PR을 등록한다
- [x] 코드가 테스트 되었나요
- [x] 문서는 업데이트 되었나요
- [x] 불필요한 코드를 제거했나요
- [x] 이슈와 라벨이 등록되었나요

## 📆 마감일

> Close #{issue number}

bsko98 committed:
feat: BaseEntity 및 Entity 코드 작성

bsko98 committed:
Merge branch 'dev' of https://github.com/devnogi/devnogi-community-server into feat/create-entity

bsko98 committed:
feat: BaseEntity 및 Entity 코드 작성

dev-ant commented on src/main/java/until/the/eternity/dcs/common/entity/BaseEntity.java:
```diff
@@ -0,0 +1,38 @@
+@Setter
+public abstract class BaseEntity {
+
+    @CreatedDate
```
@CreatedDate <-> @PrePersist 중복 사용이에요. 둘 중 하나 제거해주세요. @LastModifiedDate, @PreUpdate도 마찬가지. 둘 중에 뭘 제거했는지, 왜 둘 중에 그걸 삭제했는지 이유도 남겨주세요.

bsko98 committed:
chore: Report 엔티티 폴더 생성 및 위치 변경,
baseEntity 중복 삭제(간단한 타임스탬프 로직에는Spring Data Auditing이 더 적합하다고 판단)
그리고 EnableJpaAuditing 어노테이션 적용
